CMakeLists.txt

```diff
@@ -534,7 +534,6 @@ unset(TR_MINIUPNPC_LIBNAME)
 
 target_compile_definitions(miniupnpc::libminiupnpc
     INTERFACE
-        SYSTEM_MINIUPNP
         $<$<VERSION_LESS:${MINIUPNPC_VERSION},1.7>:MINIUPNPC_API_VERSION=${MINIUPNPC_API_VERSION}>) # API version macro was only added in 1.7
```

Comment:
Realistically, is anyone still using (or would want to use) <1.7 (released May 24, 2012)? I'd drop this as well, along with attempts to find the version in cmake/FindMINIUPNPC.cmake. If we still want to protect against <1.7, adding `#ifndef MINIUPNPC_API_VERSION / #error miniupnpc >= 1.7 is required / #endif` to libtransmission/port-forwarding-upnp.cc should be enough.

Reply:
6f75a4a That simplifies a lot of things, glad to do it.

I also removed `#if` branches for `MINIUPNPC_API_VERSION < 8`, since the `MINIUPNPC_API_VERSION` value for miniupnpc `1.7` is 8.